test/collections/VectorListTest.ooc

```diff
@@ -195,8 +243,12 @@ VectorListTest: class extends Fixture {
 			list add(16)
 			list add(32)
 			iterator := list iterator()
+			expect(iterator hasNext?(), is equal to(true))
 			for ((index, item) in iterator)
 				expect(item, is equal to(list[index]))
+			expect(iterator hasNext?(), is equal to(false))
+			secondIterator := list iterator()
```

Comment:
@thomasfanell I've seen many tests where nothing at all is freed. I thought I was just being pedantic sometimes. But you're right, will fix.